server/start.go

```diff
@@ -205,6 +221,7 @@ is performed. Note, when enabled, gRPC will also be automatically enabled.
 	cmd.Flags().Uint32(FlagStateSyncSnapshotKeepRecent, 2, "State sync snapshot to keep")
 	cmd.Flags().Bool(FlagDisableIAVLFastNode, false, "Disable fast node for IAVL tree")
 	cmd.Flags().Int(FlagMempoolMaxTxs, mempool.DefaultMaxTx, "Sets MaxTx value for the app-side mempool")
+	cmd.Flags().Uint32(FlagShutdownGraceSeconds, 0, "On Shutdown, Seconds to wait for database to flush to disk")
```

Comment:
Why not using a duration flag instead? This would clean up the logic above and make it more user friendly as way.

Reply:
ok